controller/volume_controller.go

```diff
@@ -551,6 +551,12 @@ func (vc *VolumeController) ReconcileVolumeState(v *longhorn.Volume, e *longhorn
 			return err
 		}
 		v.Spec.NodeID = usableNode.Name
+		//User should not access the front end by accident
+		e.Spec.Frontend = ""
```

Comment:
Will this field reset after the restoration completed? Otherwise we won't able to use the volume after.